### PR TITLE
Added an example for String.length

### PR DESCRIPTION
**Pull request by irenesmith**

### CODE REVIEW

File: live-examples/js-examples/string/string-length.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<pre>
+<code id="static-js">const str='Life, the universe and everything. Answer:';
+console.log(str + ' ' + str.length); // 42
+</code>
+</pre>
```
Comment (wbamberg) [inline live-examples/js-examples/string/string-length.html]: Please end files with a newline.

File: live-examples/js-examples/string/string-length.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<pre>
+<code id="static-js">const str='Life, the universe and everything. Answer:';
```
Comment (wbamberg) [inline live-examples/js-examples/string/string-length.html]: Stick with ES5 for ES5 examples (https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6) - this means using `var` here.

Comment (wbamberg) [reply live-examples/js-examples/string/string-length.html]: Also please use spaces for readability: `var str = '...`

File: live-examples/js-examples/string/string-length.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+console.log(str + ' ' + str.length); // 42
```
Comment (wbamberg) [inline live-examples/js-examples/string/string-length.html]: Please use the standard `expected output: ` format.

File: live-examples/js-examples/string/string-length.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">const str = 'Life, the universe and everything. Answer:';
```
Comment (wbamberg) [inline live-examples/js-examples/string/string-length.html]: This should be `var` not `const`.

File: live-examples/js-examples/string/string-length.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+  
+console.log(str + ' ' + str.length);
+// expected result: 42
```
Comment (wbamberg) [inline live-examples/js-examples/string/string-length.html]: "expected output", not "expected result". Sorry to be picky, but we might as well be consistent here.

Also the output is `"Life, the universe and everything. Answer: 42"`, not `42`.

**Comment by irenesmith** [comment]
You should be picky. Don't know why I said the wrong thing. Resolved both of these comments.

**Pull request merged**